src/services/web-monitoring-db.ts

```diff
@@ -72,6 +72,18 @@ export default class WebMonitoringDb {
             .then(data => parsePage(data.data));
     }
 
+    getPagesByDomains (domains: string[]): Promise<Page[]> {
+        const daysAgo = 3;
+        const dateEarlier = new Date(new Date().setDate(new Date().getDate() - daysAgo)).toISOString();
```

Comment:
Yep. Definitely just a placeholder. I will speak to analyst admin and figure out how this should be fleshed out. I'll open an issue as a reminder.